src/petals/server/handler.py

```diff
@@ -92,11 +101,20 @@ def _unpack(req: runtime_pb2.ExpertRequest) -> Iterable[runtime_pb2.Tensor]:
         assert isinstance(block_uid, str) and isinstance(metadata, dict)
         return block_uid, inputs, metadata
 
+    async def rpc_push(self, request: runtime_pb2.ExpertRequest, context: P2PContext) -> runtime_pb2.ExpertResponse:
```

Comment:
TODO: This can be stream-to-unary handler, so that the previous server doesn't have to make a new connection each time